Assets/Scripts/Manager/GameManager.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class GameManager : MonoBehaviour {
+
+    private static GameManager instance;
+
+    private GameManager()
+    {
+        if(instance == null)
+        {
+            instance = new GameManager();
```

Comment:
This will generate a endless loop since the constructor is calling it self.
Replace with "this"